qemu/tests/cfg/expose_host_mtu.cfg

```diff
@@ -0,0 +1,12 @@
+- expose_host_mtu: install setup image_copy unattended_install.cdrom
+    requires_root = yes
+    type = expose_host_mtu
+    kill_vm = yes
+    only virtio_net
+    start_vm = no
+    nic_extra_params = ",host_mtu=4000"
+    nm_stop_cmd = systemctl stop NetworkManager
```

Comment:
`service` for rhel6 please.

Reply:
The case is not supported with RHEL6, .., RHEL3.
Added the limitation in the cfg file.

Reply:
`no RHEL.6` is enough, we don't test other older distributions now.